assemblies/pentaho-solutions/src/main/resources-filtered/pentaho-solutions/system/pentaho.xml

```diff
@@ -23,6 +23,13 @@
       If true, show hint about sample users. Ultimately, should replace login-show-users-list.
     -->
     <login-show-sample-users-hint>true</login-show-sample-users-hint>
+
+	  <!--
+	  The download-roles defines user/roles that have download action permissions. It accepts a comma-separated list of roles.
+	  System admins must change what are the users/roles that have permissions for download action. If not defined here,
+	  the default value is set to 'Administrator'.
+	  -->
+	  <download-roles>Administrator</download-roles>
```

Comment:
Not a blocker, but if we can let's have this new xml block properly indented / aligned.

Reply:
It shows fine in my setup. Let me check if I can fix this.